packages/cli/templates/webcomponents/igc-ts/projects/empty/files/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"compilerOptions": {
+	  "target": "es2018",
+	  "module": "esnext",
+	  "moduleResolution": "node",
+	  "noEmitOnError": true,
+	  "lib": ["es2017", "dom"],
+	  "strict": true,
+	  "skipLibCheck": true,
+	  "esModuleInterop": false,
+	  "allowSyntheticDefaultImports": true,
+	  "experimentalDecorators": true,
+	  "importHelpers": true,
+	  "outDir": "dist",
+	  "sourceMap": true,
+	  "inlineSources": true,
+	  "rootDir": "./",
+	  "declaration": true,
+	  "incremental": true
+	},
+	"include": ["**/*.ts"]
```

Comment:
May exclude the dist folder, as there are .d.ts files generated.   
"exclude": ["dist"],